index.d.ts

```diff
@@ -1,17 +1,20 @@
-import { Dispatch, Store, Unsubscribe } from 'redux';
+import { Dispatch, Store, Unsubscribe, AnyAction } from 'redux';
 
 export function shallowEquals(a: any, b: any): boolean;
 
-interface Options {
+type Options = {
   skipInitialCall?: boolean;
   equals?: typeof shallowEquals;
 }
```

Comment:
What would be the benefit of an interface here?

Reply:
basically type and interface make not much difference here, interface might fit object typing better